behave_django/management/commands/behave.py

```diff
@@ -179,9 +297,19 @@ def get_behave_args(self, argv=sys.argv):
 
 class BehaveArgsHelper(Command):
 
+    def __init__(self, *args, **kwargs):
+        super(BehaveArgsHelper, self).__init__(self, *args, **kwargs)
+        # Let's ignore any and all output that comes out of here, all the mayor
+        # warning were already shown to the user inside the original Command,
+        # there is no need to duplicate the messages.
```

Comment:
Please, put this in a docstring.

Also, comments are almost always a sign of non-self-explanatory code.